lib/tasks/enable_features.rake

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+task enable_features: :environment do
+  GitHubClassroom.flipper[:team_managment].enable_group :staff
+  GitHubClassroom.flipper[:repo_setup].enable_group :staff
+  GitHubClassroom.flipper[:student_identifier].enable_group :staff
```

Comment:
There's one more! `:deadlines`

Reply:
Thanks ! But why isn't `:deadlines` in `feature_flags_dependency.rb` as the others ? 🤔

Reply:
Oh whoops, sorry thats my fault. The `:deadlines` flipper is still on production but the code was actually removed a long time ago. These 3 are fine :+1:

Reply:
It's alright ! Thanks for the information !